### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
 ```
 
 :::image type="content" source="images/block-removable-storage.png" alt-text="The screen depicting the blockage of the removable storage":::
+
+
+## Q&A
+1. Q: The removable storage media limitation, what is the maximum number of USBs?   
+    - A: Internally we have validate one USB group with 100,000 medias - up to 7MB size. The policy works fine in both Intune and GPO without performance issue.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:320]:
```suggestion
Internally we have validated one USB group with 100,000 media - up to 7 MB in size. The policy works fine in both Intune and GPO without performance issues.

```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 2]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
 ```
 
 :::image type="content" source="images/block-removable-storage.png" alt-text="The screen depicting the blockage of the removable storage":::
+
+
+## Q&A
+1. Q: The removable storage media limitation, what is the maximum number of USBs?   
+    - A: Internally we have validate one USB group with 100,000 medias - up to 7MB size. The policy works fine in both Intune and GPO without performance issue. 
+2. Q: The policy does not work.
+    - A: The most common no required  [Anti-malware Client Version](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide#prepare-your-endpoints). If the version is correct, then
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:322]:
```suggestion
The most common reason is there is no required [Anti-malware Client Version](/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide#prepare-your-endpoints). If the version is correct, then:

```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 3]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
...
+    - Check whether the client machine has the policy via Registry Keys - HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control\Policy Rules and HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control\Policy Groups and HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WdDevFlt\Parameters\PolicyPackage;
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:323]:
```suggestion
    - Check whether the client's device has the policy via Registry Keys  - HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control\Policy Rules, HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control\Policy Groups, and HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WdDevFlt\Parameters\PolicyPackage.
    
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 4]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
...
+2. Q: The policy does not work.
+    - A: The most common no required  [Anti-malware Client Version](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide#prepare-your-endpoints). If the version is correct, then
+    - Check whether the client machine has the policy via Registry Keys - HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control\Policy Rules and HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control\Policy Groups and HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WdDevFlt\Parameters\PolicyPackage; 
+    - If the value is not there, the most common no required  [Anti-malware Client Version](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide#prepare-your-endpoints)  or the XML file is not correct formatted, e.g. text editor may add a BOM(byte order mark 0xEF 0xBB 0xBF) at the beginning of the files which makes the xml parsing not work
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:324]:
```suggestion
    - If the value is not there, the most common reason is there is no required  [Anti-malware Client Version](/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide#prepare-your-endpoints) or the XML file is not correctly formatted, for example, the text editor might add a byte order mark (BOM) 0xEF 0xBB 0xBF at the beginning of the files which causes the XML parsing not to work.

```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 5]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
...
+    - If the value is there and if the policy is managed via Group Policy, check whether the client machine can access the policy XML path
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:325]:
```suggestion
    - If the value is there and if the policy is managed via Group Policy, check whether the client machine can access the policy XML path.
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 6]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
...
+
+
+
+
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:331]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, please commit my suggested canges. Thanks.

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 7]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
 ```
 
 :::image type="content" source="images/block-removable-storage.png" alt-text="The screen depicting the blockage of the removable storage":::
+
+
+## Q&A
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:318]:
```suggestion
## Frequently asked questions

```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 8]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
...
+1. Q: The removable storage media limitation, what is the maximum number of USBs?
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:319]:
```suggestion
**What is the removable storage media limitation for the maximum number of USBs?**

```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 9]
```diff
@@ -312,3 +313,19 @@ DeviceEvents
 ```
 
 :::image type="content" source="images/block-removable-storage.png" alt-text="The screen depicting the blockage of the removable storage":::
+
+
+## Q&A
+1. Q: The removable storage media limitation, what is the maximum number of USBs?   
+    - A: Internally we have validate one USB group with 100,000 medias - up to 7MB size. The policy works fine in both Intune and GPO without performance issue. 
+2. Q: The policy does not work.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:321]:
```suggestion
**Why does the policy not work?**

```

**Comment by yogkumgit** [comment]
@tewchen Please implement the suggested changes if you agree. Thanks

**Comment by tewchen** [comment]
one of the engineers raised security questions about exposing regkey values - HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Device Control, so i create another pull with all the updates but no regkey.

**Pull request closed without merging**